### PR TITLE
Fix datapusher woes

### PR DESCRIPTION
**Pull request by JackMc**
As discussed in pull request for datapusher

### CODE REVIEW

File: datacats/scripts/web.sh [hunk 1]
```diff
@@ -20,7 +20,7 @@ fi
 while true; do
 	# fix our development.ini
 	if [ "$2" = "True" ]; then
-		/scripts/adjust_devini.py "$(ip route get 8.8.8.8 | awk 'NR==1 {print $NF}')" "$port"
+		/scripts/adjust_devini.py "$(ip route get 8.8.8.8 | awk 'NR==1 {print $NF}')" "5000"
```
Comment (wardi) [inline datacats/scripts/web.sh]: this won't work if running with apache, right?

Comment (JackMc) [inline datacats/scripts/web.sh]: Durr

**Comment by JackMc** [comment]
My bad, fixed in latest

**Comment by wardi** [comment]
last thing we need is a regression test. Maybe just check the run/development.ini file to see if the site_url was modified properly.

**Comment by JackMc** [comment]
Hmm... Thought I added that. Maybe it's on my VM

**Comment by JackMc** [comment]
Test added - don't know if it passes though, as CI is down

**Pull request merged**